modules/porous_flow/doc/content/modules/porous_flow/tests/sinks/sinks_tests.md

```diff
@@ -413,54 +347,35 @@ \chapter{Advection of a fluid component}
 MOOSE.  This is due to numerical diffusion, which is particularly
-strong in the upwinding scheme implemented in the PorousFlow module.
+strong in the upwinding scheme used in this test.  For many more details, see the [stabilization page](stabilization.md).
 Nevertheless, MOOSE advects the smooth front with the correct
-velocity, as shown in Figure~\ref{s09.fig}.
+velocity, as shown in [s09_fig].
 
-The sharp front is {\em not} maintained by MOOSE even when no
+The sharp front is not maintained by MOOSE even when no
 upwinding is used.  In the case at hand, which uses a fully-saturated
-single-phase fluid, the {\tt FullySaturated} versions of the Kernels
+single-phase fluid, the [FullySaturated](PorousFlowFullySaturatedDarcyFlow.md) versions of the Kernels
 may be used in order to compare with the standard fully-upwinded
-Kernels.  The {\tt FullySaturated} Kernels do not employ any upwinding
+Kernels.  These Kernels do not employ any upwinding
 whatsoever, so less numerical
-diffusion is expected.  This is demonstrated in
-Figure~\ref{s09.fig}.  Two additional points may also be
+diffusion is expected.  This is demonstrated in [s09_fig].  Two additional points may also be
 nocied: (1) the lack of upwinding has produced a ``bump'' in the
 mass-fraction profile near the concentrated side; (2) the lack of upwinding
 means the temperature profile moves slightly slower than it should.
 These two affects reduce as the mesh density is increased, however.
 
-\begin{figure}[htb]
-\begin{center}
-\begin{tabular}{cc}
-\includegraphics[width=7cm]{s09_0_1.pdf}  &
-\includegraphics[width=7cm]{s09_0_5.pdf}
-\end{tabular} \\
-\includegraphics[width=7cm]{s09_1_0.pdf}
-\caption{Results of the advection of a fluid component test,
-  illustrating that the numerical implementation of porous flow within
-  MOOSE diffuses sharp fronts, but advects them at the correct
-  velocity ($v=1$\,m.s$^{-1}$ in this case, and notice the centre of
-  the front is at the correct position in each picture).  Less
-  diffusion is experienced when upwinding is not used.  Top left: mass
-  fraction at $t=0.1$\,s.  Top right: mass fraction at $t=0.5$\,s.
-  Bottom: mass fraction at $t=1.0$\,s.}
-\label{s09.fig}
-\end{center}
-\end{figure}
-
-
-
-
+The input file using the fully-saturated Kernels:
 
+!listing modules/porous_flow/test/tests/sinks/s09_fully_saturated.i
 
+!media sinks/s09_0_1.png style=width:50%;margin-left:10px
 
-\end{document}
+!media sinks/s09_0_5.png style=width:50%;margin-left:10px
 
+!media sinks/s09_1_0.png style=width:50%;margin-left:10px caption=Results of the advection of a fluid component test, illustrating that the numerical implementation of porous flow within MOOSE diffuses sharp fronts, but advects them at the correct velocity (which is 1m/s in this case).  Notice the centre of the front is at the correct position in each picture.  Less diffusion is experienced when upwinding is not used.  Top: 0.1s.  Middle: 0.5s.  Bottom: 1.0s.  id=s09_fig
```

Comment:
These figures are low resolution and really spread out. Can you make them one higher res png?